controller/src/main/java/org/jetbrains/MavenController.java

```diff
@@ -1,96 +1,146 @@
 package org.jetbrains;
 
+import ch.epfl.scala.bsp4j.*;
```

Comment:
It's usually a good practice to avoid wildcard imports, you can disable it on IDEA via File -> Settings -> Editor -> Code style -> Java and changing class and name counts to import with * to bigger value.

Reply:
Yeah, I didn't realize Intellij does this automatically, will disable that